src/main/scala/edu/berkeley/cs/rise/opaque/benchmark/BigDataBenchmark.scala

```diff
@@ -115,4 +115,12 @@ object BigDataBenchmark {
       df
     }
   }
+
+  def q3Limit(spark: SparkSession, securityLevel: SecurityLevel, size: String, numPartitions: Int)
+      : DataFrame = {
+    import spark.implicits._
+    val df = q3(spark, securityLevel, size, numPartitions)
+    df.limit(2)
```

Comment:
Given a limit of 2, how does the MultiplePartitionSuite handle this? Will 2 of the 3 partitions return 1 row, and the other return nothing?

I think it'd be good for the purpose of the test for each partition to return something, but for the partitions to not all return the same number of rows. Maybe a limit of 5?

Reply:
If we're not using this test anymore, can we remove this `q3Limit` function? Or maybe it'd be good to add this test back into OpaqueOperatorTests -- won't hurt.